### ISSUE TITLE
Meta Description - School & Program pages vs taxonomy pages

### ISSUE THREAD

**Issue by ucem** [ehodul/universities-colleges-schools#77]
Hi Erika,  I have a couple of questions which I'll post in separate Issues.  

1. We've noticed that our career pages are returned in SERP results at a much higher rate that program pages for example.  In some cases this seems odd but in other cases, it may make sense. For e.g. For the query "nursing assistant program", the /career/Nursing-Assistant page ranks high and such is the case for most queries; ideally we'd prefer a program page is ranked higher because searchers are not clicking on the career pages nearly as much as we'd like them to. Obviously based on the snippet they see, it doesn't look like what they are searching for.  I get the fact that the word "nursing assistant" is used multiple times on the page, so maybe that is it.  I know that Google's algorithms are a mystery, but given the poor job of coding on the part of the original developer, I was wondering if you know of anything on the program pages that could cause them to not be found?  

2.  To go along with #1, we've been experimenting with "Description" under Meta Tags and changes we've made in school or program pages are seen in the snippet.  However, adding a description in the career taxonomy page does not net the same result. For instance, I added a description in career taxonomy "Medical Receptionist"  on April 23, it was crawled by Google on the 25th, and the snippet doesn't reflect the changes.  Is this feature enabled in Taxonomy?

3. I recall that we talked about [node:summary] under Meta Description at one time and that if it is left as the default, and there is no content in the "Edit Summary" box, that search engines will generate a snippet from "program information".  My question is, is it best to enter information in the "Edit Summary" and leave [node:summary] under Meta Description or are they both unique and should I enter different content in each?  

Sorry for the long winded questions, hope that makes sense.

**Comment by ucem** [comment]
We noticed that the Meta Description isn't working the same for Taxonomy pages as it is for nodes.  We added a meta description for /career/Payroll and when looking at the code it still picks up all the content in Description, not what is written in Meta Description.

How long would this take to correct?  I don't want to spend too much money or time doing a lot of work since we're going to be doing the mobile responsive site, but we're looking at all avenues to see why our visitors have dropped off.
